index.html

```diff
@@ -27,17 +27,42 @@
       </p>
     </section>
     <section id="sotd">
-        <p>This is a draft for a future SocialCG report.</p>
+        <p>This is a draft of a future SocialCG report.</p>
       </section>
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-        Data portability is the ability of a user to move their data on one
+        Data portability is the ability of a user to move their data from one
         system to another system. This includes moving between ActivityPub
```

Comment:
If you're intending to define the terms "instance" and "server", do you want to leave this "system" terminology here or update it? it seems fine to me but I thought I'd mention it.

Reply:
would "from one instance to another instance, even if they are instances of different servers" be too confusing for the first sentence of an introduction? 🙃

Reply:
> If you're intending to define the terms "instance" and "server", do you want to leave this "system" terminology here or update it? it seems fine to me but I thought I'd mention it.

Yeah, good catch. I wasn't sure how strongly we felt about 'system' as a term. Maybe we can come back to it